src/weekday.ts

```diff
@@ -42,7 +42,7 @@ export class Weekday {
   toString() {
     let s: string = ALL_WEEKDAYS[this.weekday]
     if (this.n) s = (this.n > 0 ? '+' : '') + String(this.n) + s
-    return s
+    return s as WeekdayStr
```

Comment:
I don't think this is correct, the previous line will prepend the number, so it's not going to be a `WeekdayStr` which is strictly 

```ts
export type WeekdayStr = 'MO' | 'TU' | 'WE' | 'TH' | 'FR' | 'SA' | 'SU'
```

instead it outputs values like `+1TU`